routes/web.php

```diff
@@ -27,9 +27,7 @@
 
 Route::get('/', [IndexController::class, 'show'])->name('home')->middleware(['installation:complete', 'migration:complete']);
 Route::get('/gallery', [IndexController::class, 'gallery'])->name('gallery')->middleware(['installation:complete', 'migration:complete']);
-Route::match(['get', 'post'], '/migrate', [Administration\UpdateController::class, 'migrate'])
-	->name('migrate')
-	->middleware(['installation:complete', 'migration:incomplete']);
+Route::match(['get', 'post'], '/migrate', [Administration\UpdateController::class, 'migrate'])->name('migrate')->middleware(['installation:complete', 'migration:incomplete']);
```

Comment:
Has anything changed here except for line breaks and indentation again?

Reply:
nope.

Reply:
reverted.